src/chain/bls_emu.rs

```diff
@@ -70,7 +70,7 @@ impl PublicKeyShare {
 
 impl PublicKeySet {
     pub fn from_elders_info(elders_info: EldersInfo) -> Self {
-        let threshold = elders_info.members().len() * THRESHOLD_NUMERATOR / THRESHOLD_DENOMINATOR;
+        let threshold = elders_info.len() * THRESHOLD_NUMERATOR / THRESHOLD_DENOMINATOR;
```

Comment:
General comment:
Could you Add in PR description that this also address a performance issue, and any number you have.